qemu/tests/cfg/spapr_vty_multi_backends.cfg

```diff
@@ -5,21 +5,27 @@
     image_snapshot = yes
     serials = "vs1"
```

Comment:
Since you move chardev vs1~vs4 into the global variant, please also update this line.